contracts/InternalContracts/InternalContractsHandler.sol

```diff
@@ -19,13 +20,12 @@ abstract contract InternalContractsHandler is ERC1820Context {
         // Support to sponsor all users by default.
         address[] memory users = new address[](1);
         users[0] = address(0);
-        SponsorWhitelistControl(0x0888000000000000000000000000000000000001).addPrivilege(users);
+        InternalContracts.SPONSOR_CONTROL.addPrivilege(users);
 
         // remove contract admin
-        AdminControl ac = AdminControl(0x0888000000000000000000000000000000000000);
-        ac.setAdmin(address(this), address(0));
+        InternalContracts.ADMIN_CONTROL.setAdmin(address(this), address(0));
         require(
-            ac.getAdmin(address(this)) == address(0),
+            InternalContracts.ADMIN_CONTROL.getAdmin(address(this)) == address(0),
             "require admin == null"
         );
```

Comment:
If a non-admin called AdminControl's setAdmin method, this method also success but nothing will change.
So this require can make sure the admin deletion success.

Reply:
@Thegaram This is a legacy issue that full node has already fixed. Anyway, we could still keep this `require` as it is. At least, there is nearly no cost and it could prevent regression of fullnode.